server/.env.example

```diff
@@ -1,4 +1,5 @@
 STATIC_DIR=../client/html
+REVOLUT_API_URL=https://sandbox-merchant.revolut.com
```

Comment:
should we add some note in the README that merchants can try prod by changing the url and keys?

Reply:
Updated